src/util/tools.ts

```diff
@@ -376,3 +376,6 @@ export const isPartitionFullIndexSet = (
   const freeIndexSet = getFreeIndexSet(outcomesSlotCount, partition)
   return freeIndexSet === 0
 }
+
+export const capitalizeOnlyFirstLetter = (str: string) =>
+  str.charAt(0).toUpperCase() + str.slice(1).toLowerCase()
```

Comment:
This could be named `capitalize` because that is what it does.

Reply:
Done!